ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/interfaces.py

```diff
@@ -198,7 +198,10 @@ def parseIfconfig(self, ifconfig, device, relMap):
                     ip, netmask = maddr.groups()[:2]
                 else:
                     ip, netmask = maddr.groups()[2:]
-                netmask = self.maskToBits(netmask)
+                try:
+                    netmask = self.maskToBits(netmask)
+                except ValueError:
+                    return relMap
```

Comment:
`self.maskToBits(netmask)` is failing with a ValueError because netmask is `ff000000` and maskToBits is expecting netmask to be a string like `255.255.255.0` or in this case `255.0.0.0`.

Silently aborting modeling here seems extreme since ff000000 is a valid representation of a netmask that we could deal with, just not with the built-in maskToBits method.